### PR TITLE
Add -p aliases for all ? functions.

### LINKED ISSUE #64: Provide foo-p aliases of foo? commands
It's more common in elisp to use `foo-p` for predicates. Would you be willing to add these as aliases? I'm happy to write a PR.

### PR DESCRIPTION
**Pull request by Wilfred**
Fixes #64

### CODE REVIEW

File: README.md.tpl [hunk 1]
```diff
@@ -468,7 +486,7 @@ Alias: `f-dir?`
 
 {{f-same?}}
 
-Alias: `f-equal?`
+Alias: `f-same-p f-equal? f-equal-p`
```
Comment (rejeep) [inline README.md.tpl]: You typed _"Aliases"_ above when there are multiple aliases, please change this as well.

Comment (Wilfred) [inline README.md.tpl]: Fixed.

**Comment by Wilfred** [comment]
I've fixed the Alias/Aliases issue.

**Comment by rejeep** [comment]
Merged, thanks! 👍

**Pull request merged**